src/static/themes/greenmat/aldrich-ff.css

```diff
@@ -0,0 +1,6 @@
+@font-face {
+  font-family: 'Aldrich';
```

Comment:
Is this a monospaced font? If yes, we could use it for every theme since cmcaine mentioned wanting to bundle a monospaced font for windows in https://github.com/cmcaine/tridactyl/issues/94

Reply:
I'd rather we left the default as an option for the user; people should just set it in their Firefox preferences and they'll get nicer fonts everywhere.

Reply:
I think it's monospaced but didnt find confirmation: https://fonts.google.com/specimen/Aldrich